scipy/stats/stats.py

```diff
@@ -3900,6 +3900,46 @@ def ks_2samp(data1, data2):
     return d, prob
 
 
+def cramer_vonmises_2samp(x, y):
+    """
+    Computes the Cramer von Mises two sample test.
+    
+    This is a two-sided test for the null hypothesis that 2 independent samples
+    are drawn from the same continuous distribution.    
+    
+    Parameters
+    ----------
+    x, y : sequence of 1-D ndarrays
+        two arrays of sample observations assumed to be drawn from a continuous
+        distribution, sample sizes can be different
+
+    Returns
+    -------
+    T : float
+        T statistic
+    p-value : float
+        two-tailed p-value
+    """
+    # following notation of Anderson et al. doi:10.1214/aoms/1177704477
+    N = len(x)
+    M = len(y)
+    alldata = np.concatenate((x,y))
+    allranks = rankdata(alldata)
+    ri = allranks[:N]
+    sj = allranks[-M:]
+    i = rankdata(x)
+    j = rankdata(y)
+    # Anderson et al. Eqn 10
+    U = N*np.sum((ri - i)**2) + M*np.sum((sj - j)**2)
+    # Anderson et al. Eqn 9
+    T = U/(N * M * (N + M)) - (4 * M * N - 1)/(6 * (M + N))
+    Texpected = 1/6 + 1/(6 * (M + N))
+    Tvariance = 1/45 * (M + N + 1)/(M + N)**2 * (4 * M * N * (M+N) - \ 
```

Comment:
`Texpected` and `Tvariance` come out consistently as zero due to the fact that the math here is integer-based.  Need to make these two lines use floats.